paper-radio-button.html

```diff
@@ -65,27 +68,32 @@
       }
 
       #radioContainer {
-        display: inline-block;
+        @apply(--layout-inline);
+        @apply(--layout-center-center);
         position: relative;
-        width: 16px;
-        height: 16px;
+        width: var(--calculated-paper-radio-button-size);
+        height: var(--calculated-paper-radio-button-size);
         vertical-align: middle;
       }
 
       #ink {
         position: absolute;
-        top: -16px;
-        left: -16px;
-        width: 48px;
-        height: 48px;
+        top: 50%;
+        left: 50%;
+        width: calc(2.66 * var(--calculated-paper-radio-button-size));
+        height: calc(2.66 * var(--calculated-paper-radio-button-size));
         color: var(--paper-radio-button-unchecked-ink-color, --primary-text-color);
         opacity: 0.6;
         pointer-events: none;
+        -webkit-transform: translate(-50%, -50%);
+        transform: translate(-50%, -50%);
       }
 
       :host-context([dir="rtl"]) #ink {
```

Comment:
The ripple container is set to `#radioContainer` though, which is relatively positioned. I've tried playing around with a couple different ideas, but I don't think I'm seeing what you're seeing.